domain/src/main/java/edu/nextstep/camp/calculator/domain/MemoryCalculateStorage.kt

```diff
@@ -0,0 +1,11 @@
+package edu.nextstep.camp.calculator.domain
+
+class MemoryCalculateStorage(history: List<HistoryItem> = emptyList()) : CalculateStorage {
+    private val _history: MutableList<HistoryItem> = history.toMutableList()
+    override val history: List<HistoryItem>
+        get() = _history
```

Comment:
```suggestion
    override val history: List<HistoryItem>
        get() = _history.toList()
```
외부에서 Mutable로 변환해서 데이터를 변경하면 실제로 변경이 되기 때문에, 가급적이면 방어적 복사를 통해 원본과의 연결을 끊어주시는 것이 좋습니다.